docs/content/en/docs/yaml-crd-ref/evaluationdefinition.md

```diff
@@ -1,5 +1,132 @@
 ---
 title: KeptnEvaluationDefinition
-description: Define all workloads and checks associated with an application
-weight: 25
+description: Define an evaluation query
+weight: 20
 ---
+
+A `KeptnEvaluationDefinition` assigns target values
+to [KeptnMetric](metric.md) queries.
+These are used as part of evaluation tasks
+that can be run by the Keptn Lifecycle Toolkit
+as part of pre- and post-analysis phases of a workload or application.
+You can then create a [KeptnTaskDefinition](taskdefinition.md)
+that describes an action to take based on the results of the evaluation.
```

Comment:
These two lines are not true, I think they should be remove completely

Reply:
Got it!  I think the text is now correct.